src/PhpParser/Parser/ParserErrors.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\PhpParser\Parser;
+
+use PHPStan\Parser\ParserErrorsException;
+
+final class ParserErrors
+{
+    private string $message;
+
+    private int $line;
+
+    public function __construct(ParserErrorsException $exception)
+    {
+        $this->message = $exception->getMessage();
+        $this->line = $exception->getAttributes()['startLine'] ?? 0;
```

Comment:
I think it is better to have fallback for `$exception->getLine()`:

```suggestion
        $this->line = $exception->getAttributes()['startLine'] ?? $exception->getLine();
```